.eslintrc.js

```diff
@@ -99,7 +99,7 @@ module.exports = {
     'no-caller': 'warn',
     'no-catch-shadow': 'warn',
     'no-confusing-arrow': 'warn',
-    'no-console': 'error',
+    'no-console': 'off',
```

Comment:
Good call. We use `console.warn` heavily, but should probably either provide guidance on how to remove w/ a bundler, or use `debug` and turn off when built for production.